help/md/floor.md

```diff
@@ -1,9 +1,14 @@
 ## name
 floor
 ## title
+The floor function rounds a number down to the nearest integer
 ## description
+The floor function is the function that takes as input a real number x, and gives as output the greatest integer less than or equal to x. 
 ## details
+ For example, for floor: ⌊2.4⌋ = 2, ⌊−2.4⌋ = −3
 ## authors
 ## see_also
```

Comment:
For the `see_also`, can you add `trunc`?  It is similar to `floor`, but `trunc(-2.4) == -2`.